src/ui/Editor.h

```diff
@@ -248,6 +252,15 @@ class Editor : public BScintillaView {
 			BMessage			fDocumentSymbols;
 			symbols_status		fSymbolsStatus;
 			std::set<std::pair<std::string, int32> > fCollapsedSymbols;
+
+			// editorconfig
+			bool				fHasEditorConfig;
+			IndentStyle			fIndentStyle;
+			int32				fIndentSize;
+			int32				fEndOfLine;
+			bool				fTrimTrailingWhitespace;
+			// bool				fInsertFinalNewline;
+
```

Comment:
We don't have to do right now, but wouldn't it better to out all of these inside a struct, maybe allocated on the heap? I guess it could become crowded if there are more editorconfig options

Reply:
Yes, I had already planned to do it. Done.